### PR TITLE
Add poker87c and poker87d

### PR DESCRIPTION
**Pull request by mfkiiyd**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by drashna** [comment]
These look like they're were pulled straight from kbfirmware.  which is okay, but it's 4+ years out of date, and a lot has changed. 

If you have the json's from kbfirmware, then please run them through this: https://noroadsleft.github.io/kbf_qmk_converter/

And then re-upload the newer files, as they should be pretty in line with current standards and config.

**Comment by mfkiiyd** [comment]
File has been updated, please pull merge, thank you

File: keyboards/poker87c/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+void matrix_init_user(void) {
+}
+
+void matrix_scan_user(void) {
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+	return true;
+}
```
Comment (drashna) [inline keyboards/poker87c/keymaps/default/keymap.c:56]: These are not being used and should be removed. 
```suggestion
```

File: keyboards/poker87c/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+		
+	}
+
+	if (usb_led & (1 << USB_LED_KANA)) {
+		
+	} else {
+		
+	}
+
+}
```
Comment (drashna) [inline keyboards/poker87c/keymaps/default/keymap.c:90]:
```suggestion
```

Can/should be replaced with a define in the keyboard's config.h file:
```c
#define LED_NUM_LOCK_PIN B1
#define LED_CAPS_LOCK_PIN B3
#define LED_SCROLL_LOCK_PIN B2
```

File: keyboards/poker87c/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+		
+	}
+
+	if (usb_led & (1 << USB_LED_KANA)) {
+		
+	} else {
+		
+	}
+
+}
```
Comment (drashna) [inline keyboards/poker87c/keymaps/via/keymap.c:105]:
```suggestion
```

File: keyboards/poker87c/config.h [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+//     #define RGBLIGHT_EFFECT_RAINBOW_SWIRL
+//     #define RGBLIGHT_EFFECT_SNAKE
+//     #define RGBLIGHT_EFFECT_KNIGHT
+//     #define RGBLIGHT_EFFECT_CHRISTMAS
+//     #define RGBLIGHT_EFFECT_STATIC_GRADIENT
+//     #define RGBLIGHT_EFFECT_RGB_TEST
+//     #define RGBLIGHT_EFFECT_ALTERNATING
+#endif
+
+
```
Comment (drashna) [inline keyboards/poker87c/config.h:74]:
```suggestion
#define LED_NUM_LOCK_PIN B1
#define LED_CAPS_LOCK_PIN B3
#define LED_SCROLL_LOCK_PIN B2
```

File: keyboards/poker87d/config.h [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+//     #define RGBLIGHT_EFFECT_BREATHING
+//     #define RGBLIGHT_EFFECT_RAINBOW_MOOD
+//     #define RGBLIGHT_EFFECT_RAINBOW_SWIRL
+//     #define RGBLIGHT_EFFECT_SNAKE
+//     #define RGBLIGHT_EFFECT_KNIGHT
+//     #define RGBLIGHT_EFFECT_CHRISTMAS
+//     #define RGBLIGHT_EFFECT_STATIC_GRADIENT
+//     #define RGBLIGHT_EFFECT_RGB_TEST
+//     #define RGBLIGHT_EFFECT_ALTERNATING
+#endif
```
Comment (drashna) [inline keyboards/poker87d/config.h:72]:
```suggestion
#endif

#define LED_NUM_LOCK_PIN B1
#define LED_CAPS_LOCK_PIN B3
#define LED_SCROLL_LOCK_PIN B2
```

File: keyboards/poker87d/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+		
+	}
+
+	if (usb_led & (1 << USB_LED_KANA)) {
+		
+	} else {
+		
+	}
+
+}
```
Comment (drashna) [inline keyboards/poker87d/keymaps/default/keymap.c:91]:
```suggestion
```

File: keyboards/poker87d/keymaps/via/keymap.c [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+		
+	}
+
+	if (usb_led & (1 << USB_LED_KANA)) {
+		
+	} else {
+		
+	}
+
+}
```
Comment (drashna) [inline keyboards/poker87d/keymaps/via/keymap.c:109]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by mfkiiyd — COMMENTED** [review]
thank you

**Review by drashna — APPROVED** [review]

File: keyboards/poker87c/config.h [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Hotswap
+#define PRODUCT         POKER-87
+#define DESCRIPTION     Keyboard
```
Comment (fauxpark) [inline keyboards/poker87c/config.h:27]:
```suggestion
```

File: keyboards/poker87d/config.h [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Hotswap
+#define PRODUCT         POKER-87
+#define DESCRIPTION     Keyboard
```
Comment (fauxpark) [inline keyboards/poker87d/config.h:27]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**